include/zephyr/bluetooth/hci.h

```diff
@@ -97,6 +97,15 @@ int bt_hci_get_conn_handle(const struct bt_conn *conn, uint16_t *conn_handle);
  */
 int bt_hci_get_adv_handle(const struct bt_le_ext_adv *adv, uint8_t *adv_handle);
 
+/** @brief Get periodic advertising sync handle.
+ *
+ * @param sync Periodic advertising sync set.
+ * @param sync_handle Place to store the periodic advertising sync handle.
+ *
+ * @return 0 on success or negative error value on failure.
```

Comment:
Could consider using `@retval` to provide the possible error values for a better UX :)

Reply:
You are right, I'll try to make a PR next to also update `get_adv_handle` and `get_conn_handle`, so it add a bit of consistency